ck.sh

```diff
@@ -52,10 +53,13 @@ case "$COMMAND" in
 
 		case $yn in
 			[Yy]*)
-				cd ckeditor-dev
-
 				# Copy custom skin to ckeditor-dev
-				cp -r ../skins/moono-lexicon skins/moono-lexicon
+				cp -r skins/moono-lexicon ckeditor-dev/skins/moono-lexicon
+
+				# Convert svg icons to png
+				node svg-to-png.js
```

Comment:
I'd move this file out of the repo root, into `support/` (just to reduce top-level clutter).

I'd also name it with camelCase, to match the other script in there (`checkSubmodule.js`). (In case you're wondering, `build-config.js` is the name of the config file you get from the CKEditor website when you download a config, so we didn't change that one.)

Reply:
Roger Roger